src/docs/product/issues/performance-issues/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Performance Issues"
+sidebar_order: 15
+description: "Learn more about Performance Issues and how to use them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+The **Issue Details** page for performance issues helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific transaction event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue.
+
+<Note>
+
+You can find performance issues by searching `issue.category:performance` in **Issues**, **Discover** and **Dashboards**.
+
+</Note>
+
```

Comment:
I've moved the note to below the image so that the image follows the description of the image.
```suggestion
```